flink-tests/src/test/java/org/apache/flink/test/checkpointing/SnapshotFileMergingCompatibilityITCase.java

```diff
@@ -344,13 +343,14 @@ private static void verifyCheckpointExistOrWaitDeleted(
     }
 
     /**
-     * Traverse the checkpoint metadata and verify all the state handle is disposed.
+     * Traverse the checkpoint metadata and verify all the state handle exist or be disposed.
      *
      * @param metadata the metadata to traverse.
+     * @param exist all corresponding files of the state handle should exist
      * @return true if all corresponding files are deleted.
      */
-    private static boolean verifyCheckpointDisposed(CheckpointMetadata metadata) {
-        AtomicBoolean disposed = new AtomicBoolean(true);
+    private boolean verifyCheckpointExist(CheckpointMetadata metadata, boolean exist) {
```

Comment:
Why remove the `static` mark?

Reply:
Cause the logger is referenced [here](https://github.com/apache/flink/pull/25116/files/a2b0a3795011a8fae37480af5952e7bfface6f4a#diff-ecfc97da093a973dee815155ff11b431a45cca9853f77aab00916ad68757510dR381-R382), which is a non-static member.